src/components/JobTable/NextRun.js

```diff
@@ -1,22 +1,34 @@
 import moment from 'moment'
+import { useTimeZoneConversion } from '@dhis2/app-runtime'
 import { PropTypes } from '@dhis2/prop-types'
 import i18n from '@dhis2/d2-i18n'
+import { Tooltip } from '@dhis2/ui'
+import React from 'react'
+
+const formatDate = (dhis2Date) =>
+    `${dhis2Date
+        .getServerZonedISOString()
+        .substring(0, 19)
+        .split('T')
+        .join(' ')} (${dhis2Date.serverTimezone})`
 
 const NextRun = ({ nextExecutionTime, enabled }) => {
+    const { fromServerDate } = useTimeZoneConversion()
+
     if (!enabled || !nextExecutionTime) {
         return '-'
     }
 
-    const now = moment(Date.now())
+    const now = Date.now()
 
     /**
      * The recommendation is to run dhis2 on a server set to UTC time.
      * In that case this timestamp is also UTC. If those recommendations
      * weren't followed the time could be off, but there's nothing
      * we can do to detect that.
      */
-    const nextRun = moment.utc(nextExecutionTime)
-    const nextRunIsInPast = nextRun.isSameOrBefore(now, 'minute')
+    const nextRun = fromServerDate(nextExecutionTime)
+    const nextRunIsInPast = nextRun.getTime() <= now
```

Comment:
Is this comment still accurate now that we're converting between timezones? It should correct for a server set in a different TZ right?

Reply:
What I'm saying is, this could be removed, right?

> If those recommendations
>     * weren't followed the time could be off, but there's nothing
>     * we can do to detect that.

Since we're now detecting TZ differences and correcting for them. Or not?

Reply:
Yes; going to remove 👍 (when I get around to updating the PR)

Reply:
Updated the comment to say `Adjust for client/sever time zone difference.` (rather than say we cannot detect it)